staging/src/k8s.io/apiserver/pkg/endpoints/handlers/helpers.go

```diff
@@ -73,3 +74,17 @@ func (lazy *lazyAccept) String() string {
 
 	return "unknown"
 }
+
+// lazyAuditID implements String() string and it will retrieve the audit ID
+// (associated with the request) lazily only when required.
```

Comment:
Suggestion:

lazyAuditID implements Stringer interface to lazily retrieve the audit ID associated with the request